manim/utils/config_utils.py

```diff
@@ -146,29 +144,19 @@ def _parse_cli(arg_list, input=True):
     if input:
         # If the only command is `manim`, we want both subcommands like `cfg`
         # and mandatory positional arguments like `file` to show up in the help section.
-        if len(sys.argv) == min_argvs - 1 or _subcommands_exist():
+        only_manim = len(sys.argv) == 1
```

Comment:
Question: did you try this with both `python -m manim` and `manim` and `manimcm`?

Reply:
Yes. In the case of `python -m manim`, sys.argv only contained [`-m`]. In the case of `manim` and `manimcm`, the only command was `manim` or `manimcm`.